spec/unit/pops/validator/validator_spec.rb

```diff
@@ -128,19 +86,25 @@ def validate(model)
       expect(acceptor.error_count).to eql(0)
       expect(acceptor).to_not have_issue(Puppet::Pops::Issues::DUPLICATE_KEY)
     end
+  end
 
-    it 'does not produce an error for duplicate default in a case expression' do
-      acceptor = validate(parse('case 1 { default: {1} default : {2} }'))
-      expect(acceptor.warning_count).to eql(0)
-      expect(acceptor.error_count).to eql(0)
-      expect(acceptor).to_not have_issue(Puppet::Pops::Issues::DUPLICATE_DEFAULT)
-    end
+  [:off, :warning, :error].each do |level|
```

Comment:
Because it shows that the change has effect - ie. this is a transition from being controlled by strict to not being controlled by strict.
If I remove that check there would be no assertion that this PR makes the required change.
Long term it is wasteful to keep that.